### ISSUE TITLE
No. is blocked, call monitor stays open

### ISSUE THREAD

**Issue by Christian-CFP** [SeydX/homebridge-fritz-platform#246]
If a number is blocked in the fritzbox and the number is calling the call monitor stay open.
This happens only for blocked numbers.

**Comment by smartmatic** [comment]
I have the same issue. Problem is now that i have an open sensor in Apple Home which is still open. Therefore my Automation does not work, as the sensor shows still open. This sensor set my TV on mute when a call is incoming.

How can i close the sensor manually?

**Comment by Christian-CFP** [comment]
The only way is to make your self a phone call (just for 1 sec) to reset the monitor. :(

**Comment by smartmatic** [comment]
Ok, no smart but a workaround, thanks!

**Comment by Christian-CFP** [comment]
Thats not a workaround. I got the same automation like you, even to pause Apple TV if a movie is been watched. But I deactivated all those automations that are using the call monitor...

**Comment by smartmatic** [comment]
You are right, workaround means the automation can be further used which is not possible in this case. I hope @seydx can offer a solution for that issue?

**Comment by Christian-CFP** [comment]
Hey @seydx, kann mir vorstellen, dass es in dieser Zeit nicht so passt, jedoch wollte ich mal dezent anfragen, ob und wann (ca.) wir mit einem Fix rechnen können?

LG
